includes/object-cache.php

```diff
@@ -553,9 +561,6 @@ public function __construct( $fail_gracefully = true ) {
                 case 'relay':
                     $this->connect_using_relay( $parameters );
                     break;
-                case 'credis':
-                    $this->connect_using_credis( $parameters );
-                    break;
```

Comment:
Instead of harshly removing support, let use `_doing_it_wrong()` to indicate that `Credis` and `HHVM` support will be dropped in a future version to see the resistance of users first.

https://developer.wordpress.org/reference/functions/_doing_it_wrong/